### ISSUE TITLE
Awaitable attributes on message

### ISSUE THREAD

**Issue by johman10** [youtype/mypy_boto3_builder#170]
**Describe the bug**
It seems like a bunch of attributes on the message are `Awaitable[str]`. The examples I found are `body` and `message_id`, but I believe there are more. The library `types-aiobotocore` marks those keys as string though, so it results in mypy errors.

I thought this issue had overlap with https://github.com/youtype/mypy_boto3_builder/issues/168, but the fix that solved it for OP on that issue does not fix it for me, so perhaps they are different after all.

**To Reproduce**

1. Install `types-aiobotocore[essential]@2.4.0.post1`
2. Run `mypy` on the following code sample

```py
from types_aiobotocore_sqs.service_resource import Message

async def process_message(message: Message):
    body_string = await message.body  # this is correct code
```

**Actual output**

```
(error: Incompatible types in "await" (actual type "str", expected type "Awaitable[Any]"))
```

**Expected output**
No errors

**Additional context**
Linux
aioboto3@10.1.0
Poetry

**Comment by vemel** [comment]
Hello!

Thank you for the report. 

Yes, this issue is directly related to #168, and it was fixed in `types-aiobotocore 2.4.0.post1`. Please make sure you are using the latest version.

```bash
pip install -U `types-aiobotocore[essential]`
```

Looks like you still have `types-aiobotocore-sqs 2.4.0` installed, so please update to `2.4.0.post1` and let me know if it helps.

I have tested your code sample on `types-aiobotocore[essential]==2.4.0.post1`, looks good.

**Comment by johman10** [comment]
Ok, that's weird. Everything in the pyproject and poetry.lock file seemed to point to the post1 version, but the actual installed version was still 2.4.0 base.

After a full removal of the package (`poetry remove types-aiobotocore`) and a new install (`poetry add 'types-aiobotocore[essential,config]'`) it's actually working as expected.

Thank you for the support and the great package! :tada:
